Formula/snap-telemetry.rb

```diff
@@ -0,0 +1,38 @@
+class SnapTelemetry < Formula
+  desc "Snap is an opensource telemetry framework"
+  homepage "http://snap-telemetry.io/"
+  url "https://github.com/intelsdi-x/snap/archive/v0.16.1-beta.tar.gz"
+  version "0.16.1"
+  sha256 "188b526860715b48230017b7b06d33152ea9bfac163ede2604209ca41c8f1239"
+
+  head "https://github.com/intelsdi-x/snap.git"
+
+  depends_on "go"
+  depends_on "glide"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+
+    snapd = buildpath/"src/github.com/intelsdi-x/snap"
+    snapd.install buildpath.children
+
+    cd snapd do
+      system "glide", "install"
+      system "go", "build", "-o", "snapd", "-ldflags", "-w -X main.gitversion=#{version}"
+      bin.install "snapd"
+    end
+
+    snapctl = buildpath/"src/github.com/intelsdi-x/snap/cmd/snapctl"
+    cd snapctl do
+      system "go", "build", "-o", "snapctl", "-ldflags", "-w -X main.gitversion=#{version}"
+      bin.install "snapctl"
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/snapd --version")
+    assert_match version.to_s, shell_output("#{bin}/snapctl --version")
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!